services/common/src/components/projectSummary/LegalLandOwnerInformation.tsx

```diff
@@ -1,31 +1,63 @@
-import React, { FC } from "react";
+import React, { FC, useEffect, useState } from "react";
 import { Col, Row, Typography } from "antd";
-import { Field, getFormValues } from "redux-form";
+import { Field, getFormValues, getFormSyncErrors } from "redux-form";
 import RenderRadioButtons from "@mds/common/components/forms/RenderRadioButtons";
 import {
   email,
   phoneNumber,
   required,
   requiredRadioButton,
   maxLength,
+  lat,
+  lon,
+  max,
+  min,
 } from "@mds/common/redux/utils/Validate";
 import { useSelector } from "react-redux";
 import { FORM } from "@mds/common/constants";
 import RenderField from "../forms/RenderField";
 import { getDropdownMunicipalities } from "@mds/common/redux/selectors/staticContentSelectors";
 import RenderSelect from "@mds/common/components/forms/RenderSelect";
 import { normalizePhone } from "@mds/common/redux/utils/helpers";
+import CoreMap from "../common/Map";
+import RenderAutoSizeField from "../forms/RenderAutoSizeField";
 
 export const LegalLandOwnerInformation: FC = () => {
   const formValues = useSelector(getFormValues(FORM.ADD_EDIT_PROJECT_SUMMARY));
+  const formErrors = useSelector(getFormSyncErrors(FORM.ADD_EDIT_PROJECT_SUMMARY));
 
-  const { is_legal_land_owner = false } = formValues;
+  const {
+    is_legal_land_owner = false,
+    facility_coords_source,
+    facility_latitude,
+    facility_longitude,
+    legal_land_desc,
+    facility_pid_pin_crown_file_no,
+  } = formValues;
 
+  const [pin, setPin] = useState<Array<string>>([]);
   const municipalityOptions = useSelector(getDropdownMunicipalities);
 
+  const dataSourceOptions = [
```

Comment:
A lot of the work in this file is just adding in the code that supports the fields that we moved in from the Facility section.